cypress/integration/cypressActions.spec.js

```diff
@@ -1,11 +1,47 @@
 /// <reference types='cypress' />
 
-describe('Name of the group', () => {
+// const { contains } = require("cypress/types/jquery");
+
+describe('User can', () => {
+  let user;
+  let date = '03 Mar 1989';
+
   before(() => {
-    
+      cy.task('freshUser').then((freshUser) => {
+          user = freshUser;
+      });
+
+      cy.visit('/');
   });
 
-  it('', () => {
-    
+  it('submit Student registration form', () => {
+    cy.get('#firstName').type(user.userFirstName).should('have.value', user.userFirstName);
+    cy.get('#lastName').type(user.userLastName).should('have.value', user.userLastName);
+    cy.get('#userEmail').type(user.email).should('have.value', user.email);
+    cy.get('#gender-radio-1').click({force: true});
+    cy.get('.custom-control-label').contains('Female').click();
+    cy.get('#userNumber').type(user.mobileNumber).should('have.value', user.mobileNumber);
+    cy.get('#dateOfBirthInput').type('{selectall}').type(date).should('have.value', date);
+    cy.get('select.react-datepicker__month-select').select('April').should('have.value', '3');
+    cy.get('select.react-datepicker__year-select').select('1991').should('have.value', '1991');
+    cy.get('.react-datepicker__day--028').click();
+    cy.get('#subjectsContainer').type('che{enter}')
+    cy.get('.css-12jo7m5').should('contain', 'Chemistry');
+    cy.get('#hobbies-checkbox-1').click({force: true});
+    cy.get('.custom-control-label').contains('Reading').click({force: true});
+    cy.get('#currentAddress').type(user.address).should('have.value', user.address);
+    cy.get('#state').type('har{enter}').should('contain', 'Haryana');
+    cy.get('#city').type('a{enter}').should('contain', 'Karnal');
+    cy.get('#submit').click();
+    cy.get('[class="table table-dark table-striped table-bordered table-hover"]')
```

Comment:
```suggestion
    cy.get('.table.table-dark.table-striped.table-bordered.table-hover')
```

or just 
cy.get('.table-responsive')

Reply:
Thanks! The class name was corrected.